src/Synth/LFO.cpp

```diff
@@ -21,6 +21,8 @@
 
 namespace zyn {
 
+#define LIMIT(x,lu, ll) (x>lu ? lu : (x<ll ? ll : x) )
```

Comment:
LFO.cpp looks like it already includes Util.h. Util.h defines T limit<T>(T val, T min, T max). It seems like for consistency it should be used instead of another definition.